src/tile.js

```diff
@@ -154,11 +154,12 @@ $.Tile = function(level, x, y, bounds, exists, url, context2D) {
      */
     this.opacity    = null;
     /**
-     * The distance of this tile to the viewport center.
-     * @member {Number} distance
+     * The squared distance of this tile to the viewport center.
+     * Use for comparing tiles.
+     * @member {Number} squaredDistance
      * @memberof OpenSeadragon.Tile#
      */
-    this.distance   = null;
+    this.squaredDistance   = null;
```

Comment:
This is technically a breaking change, since the `distance` property is publicly documented, but presumably it's not something people actually use, so I'm sure it'll be fine.

For that matter, let's document this new property with `//` comments so it's not part of the public documentation... it doesn't seem like something we need to expose as part of the official API.